src/components/calcite-block-section/calcite-block-section.tsx

```diff
@@ -131,16 +131,17 @@ export class CalciteBlockSection {
           />
         </label>
       ) : (
-        <calcite-action
+        <label
```

Comment:
Should this have some kind of a role now since it's not using the `calcite-action`? Would `label` or `button` be better semantically?

Reply:
I can change that.
I might go with role over making it a button in order to avoid a bunch of style overrides.

Reply:
Yes, a `<button>` would be best if possible.

Reply:
I see the truth of it.